packages/polkadot/src/peoplePolkadot.poi.e2e.test.ts

```diff
@@ -0,0 +1,201 @@
+/**
+ * E2E test for Proof of Ink (PoI) personhood proving process
+ */
+
+import { sendTransaction } from '@acala-network/chopsticks-testing'
+
+import { defaultAccounts } from '@e2e-test/networks'
+import { peoplePolkadot } from '@e2e-test/networks/chains'
+import { setupNetworks } from '@e2e-test/shared'
+
+import { describe, expect, test } from 'vitest'
+
+import { MIGRATION_CONSTANTS } from './helpers/migration-constants.js'
+import { TEST_PUBLIC_KEY, TEST_VOUCHER_KEY_1, TEST_VOUCHER_KEY_2, TEST_VRF_SIGNATURE } from './helpers/test-keys.js'
+
+describe('People Polkadot PoI E2E', () => {
+  test('candidate proves personhood via proof of ink flow', async () => {
+    const [peopleClient] = await setupNetworks(peoplePolkadot)
+
+    // Just for the storage initialization to go through
+    for (let i = 0; i < 15; i++) {
+      await peopleClient.dev.newBlock()
+    }
+
+    await setupProofOfInkDesignFamily(peopleClient)
+
+    const candidate = defaultAccounts.keyring.addFromUri('//TestCandidate')
+    await fundAccount(peopleClient, candidate.address)
+
+    // Step 1: Candidate applies for proof of ink
+    console.log('Step 1: Candidate applies for Proof of Ink')
+    const applyTx = peopleClient.api.tx.proofOfInk.apply()
+    await sendTransaction(applyTx.signAsync(candidate))
+    await peopleClient.dev.newBlock()
+
+    const candidateInfo = await peopleClient.api.query.proofOfInk.candidates(candidate.address)
+    expect(candidateInfo.isSome).toBe(true)
+
+    console.log('Step 2: Candidates commits to a tattoo design')
+    const commitTx = peopleClient.api.tx.proofOfInk.commit({ DesignedElective: [0, 0] }, null)
+    await sendTransaction(commitTx.signAsync(candidate))
+    await peopleClient.dev.newBlock()
+
+    const postCommitInfo = await peopleClient.api.query.proofOfInk.candidates(candidate.address)
+    expect(postCommitInfo.isSome).toBe(true)
+
+    console.log('Step 3: Candidate submits evidence')
+    const evidenceHash = new Uint8Array(32).fill(1)
+    const submitEvidenceTx = peopleClient.api.tx.proofOfInk.submitEvidence(evidenceHash)
+    await sendTransaction(submitEvidenceTx.signAsync(candidate))
+    // Wait for evidence submission to trigger mob rule case
+    for (let i = 0; i < 3; i++) {
+      await peopleClient.dev.newBlock()
+    }
+
+    const caseCount = await peopleClient.api.query.mobRule.caseCount()
+    expect(caseCount.toNumber()).toBeGreaterThan(0)
+
+    console.log('Step 4: Evidence validation')
+    const latestCaseIndex = caseCount.toNumber() - 1
+
+    await fundAccount(peopleClient, defaultAccounts.alice.address)
+
+    const interveneTx = peopleClient.api.tx.mobRule.intervene(latestCaseIndex, { Truth: { True: null } })
+    const sudoInterveneTx = peopleClient.api.tx.sudo.sudo(interveneTx)
```

Comment:
Did you need Alice as `sudo` to be able to submit this extrinsic with a root origin?

Normally, this would be done by injecting the desired call/origin into the scheduler pallet's storage for execution next block, but the People chain does not have that pallet available (unless PoP changes that).

Alternatively, you can send an XCM `Transact` from the relay chain to the people chain containing this `mobRule.intervene` extrinsic.
Check out `sendXcmFromRelayToPeople` in `packages/shared/src/people.ts`.